roles/ceph-mds/tasks/docker/pre_requisite.yml

```diff
@@ -19,6 +19,11 @@
     state: present
   when: ansible_os_family == 'RedHat'
 
+- name: install six
```

Comment:
You can re-use this: https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-common/tasks/misc/system_tuning.yml#L19-L26

And get the vars like this:
https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-common/defaults/main.yml#L293-L298

So you should be able to have a single task with this.